shared/src/test/formationFactories.ts

```diff
@@ -139,11 +139,28 @@ export const generateBusinessNameAvailability = (overrides: Partial<NameAvailabi
     "SPECIAL_CHARACTER_ERROR",
     "UNAVAILABLE",
     "RESTRICTED_ERROR",
-    undefined,
   ];
   return {
-    similarNames: [`random-name-${randomInt()}`],
-    status: statusValues[randomIntFromInterval("0", "5")],
+    similarNames: [`some-name-${randomInt()}`],
+    status: randomElementFromArray(statusValues),
+    lastUpdatedTimeStamp: getCurrentDateISOString(),
+    ...overrides,
```

Comment:
This could reduce duplication by using `...generateBusinessNameAvailabilityResponse` if you want